git_machete/generated_docs.py

```diff
@@ -0,0 +1,938 @@
+        * `GITHUB_TOKEN` env var,
+        * content of the `.github-token` file in the home directory (`~`),
+        * current auth token from the `gh` GitHub CLI,
+        * current auth token from the `hub` GitHub CLI.Note
+
+        GitHub API server URL will be inferred from `git remote`.
+        You can override this by setting the following git config keys:
+           Remote name
+              E.g. `machete.github.remote` = `origin`
+           Organization name
+              E.g. `machete.github.organization` = `VirtusLab`
+           Repository name
+              E.g. `machete.github.repository` = `git-machete`
+
+        To do this, run `git config --local --edit` and add the following section:
+        <dim>
+          [machete "github"]
+              organization = <organization_name>
+              repository = <repo_name>
+              remote = <remote_name>
+        </dim>
+
+        <b>Options:</b>
+           <b>-c</b>, <b>--checkout-my-github-prs</b>
+              Checkout your open PRs into local branches.
+           <b>-y</b>, <b>--yes</b>
+              Don't ask for confirmation when deleting branches from git.
+
+        <b>Environment variables:</b>
+           `GITHUB_TOKEN`
+              GitHub API token.
+   """,
+    "config": """
+        Documentation about available `git machete` config keys and environment variables that change the command's default behavior.
+
+        Note: `config` is not a command as such, just a help topic (there is no `git machete config` command).
+
+        <b>Config keys:</b>
+           `machete.github.{remote,organization,repository}`:
+               When executing `git machete github <subcommand>` command, GitHub API server URL will be inferred from `git remote`.
+
+              Note
+
+              GitHub API server URL will be inferred from `git remote`.
+              You can override this by setting the following git config keys:
+
+                 Remote name
+              E.g. `machete.github.remote` = `origin`
+
+                 Organization name
+              E.g. `machete.github.organization` = `VirtusLab`
+
+                 Repository name
+              E.g. `machete.github.repository` = `git-machete`
+
+              To do this, run `git config --local --edit` and add the following section:
+
+              [machete "github"]
+                  organization = <organization_name>
+                  repository = <repo_name>
+                  remote = <remote_name>
+
+           `machete.overrideForkPoint.<branch>.{to,whileDescendantOf}`:
+              Executing `git machete fork-point --override-to=<revision> [<branch>]` sets up a fork point override for <branch>.
+              The override data is stored under `machete.overrideForkPoint.<branch>.to` and `machete.overrideForkPoint.<branch>.whileDescendantOf` git config keys.
```

Comment:
A newline before the next config key would be useful

Reply:
Actually, a new line right after the config key (and before its description) would also be _pog_

Reply:
@PawelLipski Okay, so adding the new line before the next config key can be done but not before its description.